packages/account_sdk/src/tests/account/storage.rs

```diff
@@ -0,0 +1,42 @@
+#[cfg(test)]
```

Comment:
Lets move this to a `_test.rs` file next to storage rather than this tests folder